localMatchApp/headers/FunctionsDatabase.h

```diff
@@ -0,0 +1,52 @@
+#ifndef FUNCTIONSDATABASE_H
+#define FUNCTIONSDATABASE_H
+
+#include "GeneralEvent.h"
+
+/*!
+ * The class of database functions is created to
+ * be able to develop the functions
+ * that will manage the database
+ */
+class FunctionsDatabase {
```

Comment:
This name is redundant. A class is composed of functions and variables. Think of something different, like DataBaseManager, etc